docs/sphinx/userguide/sync_rfidcards.rst

```diff
@@ -0,0 +1,71 @@
+Syncronisation RFID Cards
+*************************
+
+This components handles the synchronisation of RFID cards (audiofolder and card database entries).
+
+It allows to manage the card database entries and audiofiles of one to many Phonieboxes
+in a central place (e.g. NAS, one main Phoniebox, ...) in the network,
+but keeps the possibility to play audio offline once the data where synced.
+The synchronisation can be initiated as a "sync-all" command
+and optionally on every RFID scan for the particular CardID (and corresponding audiofolder).
+For the "sync-all" bind a cardId to the command.
+For the "RFID scan sync" feature, activate the option in the configuration
+or bind a cardId to the command for dynamic de-/activation.
+
+Synchronisation
+---------------
+
+The synchronisation will be FROM a server TO the Phoniebox, overriding existing files.
+So configuration made locally will be lost after sync.
+If you want to make the initial setup e.g. via WebUi copy the files and use it as a base for the server.
```

Comment:
I didn't quite get what you are trying to say here :-)
Maybe you can break it into 2 sentences.

Reply:
It was just a hint if someone wants to make the initial configuration of rfidcards or audiofolder via webui, there is no way to sync them TO the remote server.
These files need to be copied manually.

Maybe rephrase to:
The synchronisation will be only TO the Phoniebox, overriding existing files and configuration if present.